day_one/index.js

```diff
@@ -0,0 +1,32 @@
+const testData = require("./test_input.js");
+
+const problemOne = () => {
+  const fuelRequired = testData.map(mass => calculateFuelRequired(mass));
+  return fuelRequired.reduce((prev, cur) => prev + cur);
+};
+
+const problemTwo = () => {
+  const fuelRequired = testData.map(mass => calculateTotalFuel(mass));
+  return fuelRequired.reduce((prev, cur) => prev + cur);
+};
+
+const calculateTotalFuel = mass => {
+  const fuelRequired = calculateFuelRequired(mass);
+  if (fuelRequired <= 0) {
+    return 0;
+  }
+
+  return fuelRequired + calculateTotalFuel(fuelRequired);
+};
+
+const calculateFuelRequired = mass => Math.floor(mass / 3) - 2;
```

Comment:
i did this too but when i saw other people i noticed that they relied on the fact that doing division with integers in some languages will drop the remainder as a sort of built in round. i prefer what you did for clarity though.

Reply:
Totally. Division is complicated in Python, too. Python allows you to do `/` `//` to do different types of division.